storio-sample-app/src/main/java/com/pushtorefresh/storio/sample/db/resolvers/PersonDeleteResolver.java

```diff
@@ -0,0 +1,38 @@
+package com.pushtorefresh.storio.sample.db.resolvers;
+
+import android.support.annotation.NonNull;
+
+import com.pushtorefresh.storio.sample.db.entities.Person;
+import com.pushtorefresh.storio.sample.db.tables.CarsTable;
+import com.pushtorefresh.storio.sample.db.tables.PersonsTable;
+import com.pushtorefresh.storio.sqlite.StorIOSQLite;
+import com.pushtorefresh.storio.sqlite.operations.delete.DeleteResolver;
+import com.pushtorefresh.storio.sqlite.operations.delete.DeleteResult;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static java.util.Arrays.asList;
+
+public class PersonDeleteResolver extends DeleteResolver<Person> {
+    @NonNull
+    @Override
+    public DeleteResult performDelete(@NonNull StorIOSQLite storIOSQLite, @NonNull Person person) {
+        // We can even reuse StorIO methods
+        storIOSQLite
+                .delete()
+
+                // TODO here I have a problem with Person-class: Is this right?
```

Comment:
1. Open a transaction via `storIOSQLite.internal.beginTransaction()`
2. Delete a `person` via `storIOSQLite.internal().delete()`
3. Delete `cars` via `storIOSQLite.delete().objects(cars)`
4. Close the transaction